tests/languages/solidity/keyword_feature.test

```diff
@@ -0,0 +1,128 @@
+_
+anonymous
+as
+assembly
+assert
+break
+calldata
+case
+constant
+constructor
+continue
+contract;
+default
+delete
+do
+else
+emit
+enum;
+event
+external
+for
+from
+function
+if
+import
+indexed
+inherited
+interface;
```

Comment:
Is the semicolon needed in the test?

Reply:
Yes because of the class name detection. We use the same trick in many other languages a well, e.g. JS and C++.